layers/+completion/ivy/funcs.el

```diff
@@ -254,27 +254,30 @@ that directory."
                    (buffer-substring-no-properties
                     (region-beginning) (region-end))
                  (thing-at-point 'symbol t))))
-    (counsel-git-grep nil input)))
+    (counsel-git-grep input)))
 
 (defun spacemacs/counsel-search-docs ()
   "Search spacemacs docs using `spacemacs/counsel-search'"
   (interactive)
   (spacemacs/counsel-search dotspacemacs-search-tools
                             nil spacemacs-docs-directory))
 
-(defun spacemacs//counsel-occur ()
+(defun spacemacs//counsel-occur (&optional candidates)
   "Generate a custom occur buffer for `counsel-git-grep'."
   (ivy-occur-grep-mode)
   (setq default-directory (ivy-state-directory ivy-last))
-  (let ((cands ivy--old-cands))
+  (let ((cands (if (boundp 'candidates)
```

Comment:
I see this PR has already been merged but this should just be `(or candidates ivy--old-cands)`, the symbol itself is always bound in this context.

Reply:
Thanks for the suggestion. Fixed: https://github.com/syl20bnr/spacemacs/commit/b837c1d39705b9e03bd9922da249e4c154023a92